bin/include/versioning

```diff
@@ -1,19 +1,24 @@
 #!/bin/bash
 
-TRAVIS_PULL_REQUEST=${TRAVIS_PULL_REQUEST-}
+TRAVIS_BRANCH=${TRAVIS_BRANCH-}
 TRAVIS_COMMIT=${TRAVIS_COMMIT-}
 COMMIT_NUMBER=$(git rev-list --first-parent --count HEAD)
 
-if [ "$TRAVIS_PULL_REQUEST" = "false" ]; then
+# https://semver.org/#semantic-versioning-200
+
+if [ "$TRAVIS_BRANCH" = "master" ]; then
   # travis is building on master
+  # unlike quarks-operator do not set a build identifier
   ARTIFACT_VERSION="v1.0.$COMMIT_NUMBER"
 
-elif [ -n "${TRAVIS_PULL_REQUEST:-}" ]; then
-  # this is running on a travis PR
-  ARTIFACT_VERSION="v0.0.$TRAVIS_PULL_REQUEST-g$TRAVIS_COMMIT"
+elif [ -n "${TRAVIS_BRANCH:-}" ]; then
+  # this is running on a travis push
+  # return pre-release version
+  sha=$(git rev-parse --short "$TRAVIS_COMMIT")
+  ARTIFACT_VERSION="v0.0.$COMMIT_NUMBER-g$sha"
 
 else
-  # this is not running in travis
+  # return pre-release version
   COMMIT=$(git describe --tags --long | awk -F - '{ print $NF }')
   ARTIFACT_VERSION="v0.0.$COMMIT_NUMBER-$COMMIT"
```

Comment:
```suggestion
  ARTIFACT_VERSION="v0.0.$COMMIT_NUMBER-g$COMMIT"
```
Nit: just for consistency

Reply:
Ah. it has it already, nevermind

Reply:
Already in there, because COMMIT is derived from `git describe` :)